src/Core.UnitTests/Notifications/NotificationServiceTests.cs

```diff
@@ -454,56 +434,13 @@ public void ShowNotification_InfoBarButtonClicked_NonCriticalException_Exception
             logger.AssertPartialOutputStringExists("this is a test");
         }
 
-        [TestMethod]
-        public void ShowNotification_InfoBarButtonClicked_CriticalException_ExceptionNotCaught()
-        {
-            var notification = CreateNotification(actions:
-                new NotificationAction("action", _ => throw new StackOverflowException("this is a test"), false)
-            );
-
-            var infoBar = new Mock<IInfoBar>();
-
-            var infoBarManager = CreateInfoBarManager(notification, infoBar.Object);
-            var logger = new TestLogger();
-
-            var testSubject = CreateTestSubject(infoBarManager.Object, logger: logger);
-            testSubject.ShowNotification(notification);
-
-            Action act = () => infoBar.Raise(x => x.ButtonClick += null, new InfoBarButtonClickedEventArgs("action"));
-
-            act.Should().ThrowExactly<StackOverflowException>().WithMessage("this is a test");
-            logger.AssertNoOutputMessages();
-        }
-
-        [TestMethod]
-        public void ShowNotification_UserClosedTheNotification_NonCriticalException_ExceptionCaught()
```

Comment:
This test is still valid, but for CloseNotification. Why completely removing it and not adapt it instead?

Reply:
It's a duplicated test